### ISSUE TITLE
Support for multiple connected accounts

### ISSUE THREAD

**Issue by valeriansaliou** [prose-im/prose-app-web#81]

**Comment by nesium** [comment]
I'm still thinking about how we should deal with cached data when logging out. The problem being, that since introducing OMEMO encryption private keys and decrypted messages may be irretrievably lost.

We've talked briefly about OMEMO keys and sessions. I think we should scope all cached data to an account so that multiple accounts can co-exist and offer a checkbox when the user tries to log out to the tune of "[ ] Permanently delete cached data from this browser. This includes messages that may not be able to be decrypted on your other devices."

**Comment by valeriansaliou** [comment]
Agreed, could be a by default hidden advanced logout options section with some default value, ie keep all keys on logout in storage. Expandable, revealing those options but very discreet.
